### ISSUE TITLE
Build failure on v3.4.2 tag with missing cmake qt_standard_project_setup command

### ISSUE THREAD

**Issue by gl-yziquel** [ProtonMail/proton-bridge#421]
## Expected Behavior

The `make build`command should successfully complete on tage v3.4.2

## Current Behavior

It fails with the following error:

```console
+ [[ linux-gnu == \d\a\r\w\i\n* ]]
+ BRIDGE_CMAKE_MACOS_OPTS=
+ cmake -DCMAKE_BUILD_TYPE=Release '-DBRIDGE_APP_FULL_NAME=Proton Mail Bridge' '-DBRIDGE_VENDOR=Proton AG' -DBRIDGE_REVISION=a80fd92018 -DBRIDGE_TAG=v3.4.2 -DBRIDGE_DSN_SENTRY= -DBRIDGE_BRIDGE_TIME= -DBRIDGE_BUILD_ENV=dev -DBRIDGE_APP_VERSION=3.4.2+git '' -G Ninja -S . -B ./cmake-build-release
-- BRIDGE_REPO_ROOT is /home/mini-me/home/cellar/proton-bridge/internal/frontend/bridge-gui/bridge-gui/../../../..
-- VCPKG_ROOT is /home/mini-me/home/cellar/proton-bridge/internal/frontend/bridge-gui/bridge-gui/../../../../extern/vcpkg
-- The CXX compiler identification is GNU 11.4.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- App name is Proton Mail Bridge
-- App vendor is Proton AG
-- Bridge version is 3.4.2+git
-- Found qmake at /usr/bin/qmake
-- Looking for C++ include pthread.h
-- Looking for C++ include pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE  
-- Found OpenGL: /usr/lib/x86_64-linux-gnu/libOpenGL.so   
-- Found WrapOpenGL: TRUE  
-- Found XKB: /usr/lib/x86_64-linux-gnu/libxkbcommon.so (found suitable version "1.4.0", minimum required is "0.5.0") 
-- Found WrapVulkanHeaders: /usr/include  
CMake Error at CMakeLists.txt:79 (qt_standard_project_setup):
  Unknown CMake command "qt_standard_project_setup".
```

## Steps to Reproduce

I am running the build on an ubuntu 22.

## Version Information

Building tag v3.4.2

## Context (Environment)

I am trying to build from source. I like building from source. I want control over the source to see if I can perform replication of Proton IMAP contents in Dovecot via proton-bridge.

**Comment by gl-yziquel** [comment]
`make build-nogui`however succeeded and is sufficient for my needs.

**Comment by LBeernaertProton** [comment]
@gl-yziquel Do you have the Qt6.3 SDK installed on your system?

**Comment by gl-yziquel** [comment]
I pulled all Qt6 dependencies that made the build system bark from ubuntu.

If this qt_standard_project_setup command is in them, I do not know where to find it. Can't find the relevant package.

But, yes, I did pull quite a number of debian Qt6 packages. For sure.

**Comment by LBeernaertProton** [comment]
According to [the docs](https://doc.qt.io/qt-6/qt-standard-project-setup.html) this command was introduced in Qt 6.3. It's possible that the packages you obtained are not the correct version.

**Comment by gl-yziquel** [comment]
The Qt6 version that is shipped on my ubuntu 22 indeed is 6.2.4+dfsg-2ubuntu1.1.

Would be nice to know which tag of the proton-bridge repo builds with Qt6 version 6.2.

**Comment by LBeernaertProton** [comment]
You need Qt 6.3 to build bridge from source. If you distro does not have this version, I'd recommend getting it from the official Qt releases.
